packages/mobile/src/send/SendConfirmation.tsx

```diff
@@ -176,34 +181,34 @@ class SendConfirmation extends React.Component<Props, State> {
     this.sendOrInvite(InviteBy.SMS)
   }
 
-  render() {
+  renderWithAsyncFee: CalculateFeeChildren = (asyncFee) => {
     const { t, appConnected, isSending } = this.props
-    const { amount, reason, recipient, recipientAddress, fee } = this.getConfirmationInput()
+    const { amount, reason, recipient, recipientAddress } = this.getConfirmationInput()
 
     const currentBalance = this.props.dollarBalance
-    const amountWithFees = new BigNumber(numeral(amount).value()).plus(
-      this.props.suggestedFeeDollars
-    )
-    const userHasEnough = amountWithFees.isLessThanOrEqualTo(currentBalance)
+    const fee = asyncFee.result && getFeeDollars(asyncFee.result)
+    const amountWithFee = new BigNumber(numeral(amount).value()).plus(fee || 0)
+    const userHasEnough = !asyncFee.loading && amountWithFee.isLessThanOrEqualTo(currentBalance)
     const { isPaymentRequest } = this.getNavParams()
     let primaryBtnInfo = {
       action: this.onSendButtonClick,
       text: t('send'),
-      disabled: isSending || !userHasEnough || !appConnected,
+      disabled: isSending || !userHasEnough || !appConnected || !!asyncFee.error,
```

Comment:
dupe logic with below - assign to a variable

Reply:
Yep 👍